wiki/Ranking_Criteria/osu!mania/de.md

```diff
@@ -1,91 +1,205 @@
+- **Vermeide die Verwendung von Chords während eines 1/6-Streams.** Jedoch ist die Verwendung von Chords am Anfang oder Ende eines Streams in Ordnung.
+- **Von der Verwendung von 1/4 Jacks wird abgeraten.** Diese können für Spieler dieses Niveaus körperlich zu anstrengend sein. Minijacks mit ausreichender Ruhezeit dazwischen können akzeptabel sein.
+- **Vermeide die Verwendung von mehr als 9 aufeinanderfolgenden Noten in einem Trill.** Die Koordination und das Gleichgewicht der Hände, die für dieses Pattern erforderlich sind, sind für Spieler auf diesem Niveau zu schwierig.
+- **Lange Noten sollten für mindestens 1/4 eines Beats gehalten werden.**
+- **Objekte, die direkt nach kurzen Slider-Velocity-Änderungen erscheinen, sollten mit der normalen Scroll-Geschwindigkeit der Beatmap erscheinen.** Um dies zu erreichen, sollten kurze Slider-Velocity-Änderungen nur in den leeren Bereichen zwischen den Noten verwendet werden und sich auf einen Abstand von 1,0x ausgleichen.
+- **Langzeitige Slider-Velocity-Änderungen sollten zwischen 0,8x und 1,10x liegen.**
+- **Auf Slider-Velocity-Gimmicks sollte man innerhalb von 1/2 Beat oder mehr reagieren können.**
 
-- **Jede Beatmap braucht mindestens 2 Schwierigkeitsgrade. Davon muss eine mindestens Easy/Normal sein. Marathons sind davon nicht betroffen**
-- **Schwierigkeitsgrade mit verschiedener zu spielender Tastenzahl müssen als unabhängiger Modus behandelt werden und benötigen jeweils einen eigenen voll Spread.** Deswegen benötigt JEDE Tastenzahl einen eigenen Spread. Als Beispiel: Wenn du 7K Maps in deine Beatmap einfügst brauchst du einen ordentlichen und vollen 7K Spread. Das gleiche gilt für alle anderen Tastenzahlen
+Zusätzliche Richtlinien für *4 key Hard* Schwierigkeiten:
 
-### Timing
+- **Vermeide Chords mit mehr als 3 Noten.** Dies gilt auch für lange Noten-Patterns, die in mehr als drei Spalten Noten enthalten.
+- **Vermeide Anchors, die aus fünf oder mehr Noten bestehen.** Obwohl sie einfach zu realisieren sind, so sind längere Anchors anstrengend.
 
-- **Es ist nicht erlaubt Rote Timing Points zu benutzen wenn sich die BPM im Lied nicht ändert.** In anderen Worten: Es ist nicht erlaubt die Fallgeschwindigkeit der Noten mit einem Roten Timing Point zu verändert. Benutzt dafür die Grünen Inherited Timing Points.
+Zusätzliche Richtlinien für *7 key Hard* Schwierigkeiten:
 
-## Richtlinien
+- **Vermeide Chords mit mehr als 4 Noten.** Dies gilt auch für lange Noten-Patterns, die in mehr als vier Spalten Noten enthalten.
+- **Vermeide Anchors, die aus fünf oder mehr Noten bestehen.** Obwohl sie einfach zu realisieren sind, so sind längere Anchors anstrengend.
+- **Halte den Chord-Abstand in einem Chordstream bei einem Intervall von 1/1 oder höher.** Alles was dichter ist, kann die Lesbarkeit beeinträchtigen.
+- **Gelegentliche Hands können in Streams verwendet werden, solange sie durch mindestens einen Takt (4 Beats) getrennt sind.
```

Comment:
As I'm not an osu!mania player myself, what are `hands`?

Reply:
Explained in the `Patterns` section: 

> **Hand**: Drei gleichzeitig gedrückte Noten, auch "Triple" genannt.

Reply:
Thanks, I've overlooked this.